ci/pipeline.yml

```diff
@@ -91,189 +91,175 @@ jobs:
+        DELETE_SERVICE: ((acceptance-test-delete-service-staging))
+    - task: acceptance-tests-dns-01
+      file: broker-src/ci/acceptance-tests.yml
+      params:
+        <<: *acceptance-tests-params-staging
+        CHALLENGE_TYPE: DNS-01
+
+- name: deploy-production
+  plan:
+  - aggregate:
+    - get: broker-src
+      passed: [acceptance-tests-staging]
+      trigger: true
+    - get: pipeline-tasks
+    - get: broker-release
+      trigger: true
+    - get: stemcell
+      trigger: true
+    - get: terraform-yaml
+      resource: terraform-yaml-production
+    - get: common
+      resource: common-production
+      trigger: true
+  - task: terraform-secrets
+    file: broker-src/ci/terraform-secrets.yml
+  - put: production-deployment
+    params:
+      stemcells:
+      - stemcell/*.tgz
+      releases:
+      - broker-release/*.tgz
+      manifest: broker-src/bosh/manifest.yml
+      vars_files:
+      - terraform-secrets/terraform.yml
+      - common/secrets.yml
+  - task: update-broker
+    file: pipeline-tasks/register-service-broker.yml
+    params:
+      <<: *cf-creds-production
+      BROKER_NAME: domains-broker
+      BROKER_URL: ((broker-url-production))
+      AUTH_USER: ((domains-broker-user-production))
+      AUTH_PASS: ((domains-broker-pass-production))
+      SERVICES: custom-domain
+  on_failure:
+    put: slack
+    params:
+      text: |
+        :x: FAILED to deploy cf-domains-service-broker on ((cf-api-url-production))
+        <$ATC_EXTERNAL_URL/teams/$BUILD_TEAM_NAME/pipelines/$BUILD_PIPELINE_NAME/jobs/$BUILD_JOB_NAME/builds/$BUILD_NAME|View build details>
+      channel: ((slack-channel))
+      username: ((slack-username))
+      icon_url: ((slack-icon-url))
+  on_success:
+    put: slack
+    params:
+      text: |
+        :white_check_mark: Successfully deployed cf-domains-service-broker on ((cf-api-url-production))
+        <$ATC_EXTERNAL_URL/teams/$BUILD_TEAM_NAME/pipelines/$BUILD_PIPELINE_NAME/jobs/$BUILD_JOB_NAME/builds/$BUILD_NAME|View build details>
+      channel: ((slack-channel))
+      username: ((slack-username))
+      icon_url: ((slack-icon-url))
+
+- name: acceptance-tests-production
+  serial: true
+  plan:
+  - get: broker-src
+    passed: [deploy-production]
+    trigger: true
+  - aggregate:
+    - task: acceptance-tests-http-01
+      file: broker-src/ci/acceptance-tests.yml
+      params: &acceptance-tests-params-production
+        <<: *cf-creds-production
+        SERVICE_NAME: custom-domain
+        PLAN_NAME: custom-domain
+        SERVICE_INSTANCE_NAME: domains-acceptance-test-%s
+        CHALLENGE_TYPE: HTTP-01
+        AWS_ACCESS_KEY_ID: ((domains-broker-access-key-id-acceptance-production))
+        AWS_SECRET_ACCESS_KEY: ((domains-broker-secret-access-key-acceptance-production))
+        HOSTED_ZONE_ID: ((hosted-zone-id-production))
+        DOMAIN: ((domain-url-production))
+        DOMAINS_TIMEOUT: ((domains-timeout))
+        CA_CERT: ((acceptance-test-ca-cert-production))
```

Comment:
A little context: we're passing the let's encrypt staging ca certificate to development tests because development is using the let's encrypt staging endpoint, which lets us avoid normal rate limits. For production, we'll need to use the let's encrypt production endpoint, and we won't have to pass a ca certificate, because the let's encrypt production ca is already trusted.